### ISSUE TITLE
Suggestion: Tailings management. 

### ISSUE THREAD

**Issue by zEverdeen** [StrangeLoopGames/EcoIssues#6714]
Suggestion: Tailings management.

Now, this might be a welcomed suggestion, and it might not be.. but that isn’t going to stop me from looking stupid..

So here it is, with nuclear waste on the way, I think it would be interesting to revise tailings disposal.

The suggestion is as follows: Obviously nuclear waste would have to be buried, and sealed off somewhere deep down, an [old ore mine perhaps](https://github.com/StrangeLoopGames/EcoIssues/issues/6389)?

I don’t think having to bury Tailings is a bad idea. Or rather, I don’t think the idea that you have to bury something is bad, but I don’t think it should be tailings. BUT I do think the downside is too large for people to get sucked into the idea in the first place, so this is what I propose instead!

Have them _“spread it out over the ground”_

Like:
![luftaufnahmen_nordseekueste_2012-05-by-raboe-478](https://user-images.githubusercontent.com/31023534/38118426-ff2619de-33ba-11e8-981b-c350652dbc25.jpg)
Author: RaBoe/Wikipedia - [Original Image ](https://commons.wikimedia.org/wiki/File:Luftaufnahmen_Nordseekueste_2012-05-by-RaBoe-478.jpg)- [License](https://creativecommons.org/licenses/by-sa/3.0/de/legalcode)

This is a Tailings Pond. Now, as you can see, it is quite large(aka the downside of using tailings), and if you were to make a similar pond, a very flat one, that would require a “containment” block/fence around it, then the materials needed for that would increase more and more, AND if you add a maintenance mechanic to the pond-fence, quite interesting it now becomes(yoda). All of a sudden, a leak isn’t just a “small area”. Containing them are more easy initially. People will not always go for mining, masonry and smithing in combination. You will be able to see large ponds in the environment like in the real life, and nuclear waste will be buried, but tailings visible. If you stack the tailings too high, it might "rupture"/leak, forcing people to cover a larger area.

Here comes the question about the Blast Furnace, is it interesting to also have them make a small amount of tailings? I am not sure, but having people “rush” a furnace, isn’t ideal, so therefore I think they should also produce something, more than just air pollution.

I have some ideas about altering air pollution, but I will keep that for later.

[I think this doable is](https://www.youtube.com/embed/XZbVLvT7qBU?end=59)

(I heard someone likes Star Wars.)

**Comment by darkshad9999** [comment]
What the heck no this would never work spreading it out overt he ground.  If you did that it would just cause further contamination.  This is an absolutely horrible idea!  I made a suggestion about tailings about making a stone or cement box that we can put the tailings into and leave it there.

http://ecoforum.strangeloopgames.com/topic/4247/smithing

Tailings should also have a half life and should not survive forever either and after a certain amount of
time should go away. 

It is also extremely unfair that smiths have the only tailings that really destroy the environment 
it is like a punishment for being a smith and that is unfair. The whole tailings idea needs to be redone 
and is totally unfair. 

Why is it that the Smiths are the only one that has a tailings that destroys the earth. 
If the devs are going to make a properly balanced game then every other profession
in the game should have some kind of tailing that they would have to deal with
and that would pollute the earth as well to balance things out. But that isn't the case. 

Making the smiths did down into the ground like this and having to create 
mines surrounded by rocks is a waste of time and is unfair that we are the only
profession that has to do this.

As I said I suggested that they make a stone box that we can put the tailings in 
that would work similar to a chest and would prevent polluting the ground 
and maybe make a bigger box that holds even more tailings made out of cement for when
you get to a higher level. 

A lot of work needs to be done with smithing and how this whole tailing things work and
the professions clearly need to be balanced out so that every profession has some
kind of a waste they have to deal with and not just smithing.
